activities/supporting-codebase-governance/customized-governance-workshop/index.md

```diff
@@ -0,0 +1,49 @@
+---
+type: Index
+---
+
+# Customized governance workshop
+
+A customized governance workshop can be used for a codebase that already has some form of governance in place, or is considering a draft proposal.
+The state of the governance may suggest what to focus the workshop on, but its strength is leveraging the governance game to socialize understanding of the governance.
+As such, it can be used to:
+
+* simply sketch out a currently well functioning mechanism
+* talk through an initial draft or proposed changes
+* surface tensions, pain points, and differing understandings
+It may also be used to talk through different scenarios to stress test the governance mechanism.
```

Comment:
Whoops

```suggestion
* simply sketch out a currently well functioning mechanism
* talk through an initial draft or proposed changes
* surface tensions, pain points, and differing understandings

It may also be used to talk through different scenarios to stress test the governance mechanism.
```